testing_app/pubspec.lock

```diff
@@ -503,5 +503,5 @@ packages:
     source: hosted
     version: "2.2.1"
 sdks:
-  dart: ">=2.7.0 <3.0.0"
-  flutter: ">=1.16.0"
+  dart: ">=2.10.0-0.0.dev <2.10.0"
```

Comment:
What channel were you using when you updated this file?

Reply:
master. I pushed a new commit using stable.

Reply:
Actually I wonder why `pubspec.lock` is not ignored in git in this repo?